apps/web/src/middleware.ts

```diff
@@ -1,25 +1,24 @@
 import { NextRequest, NextResponse } from 'next/server';
 
-export default function middleware(req: NextRequest) {
+import { getToken } from 'next-auth/jwt';
+
+export default async function middleware(req: NextRequest) {
   if (req.nextUrl.pathname === '/') {
-    const redirectUrl = new URL('/documents', req.url);
+    const redirectUrl = new URL('/dashboard', req.url);
 
     return NextResponse.redirect(redirectUrl);
   }
 
-  // if (req.nextUrl.pathname.startsWith('/dashboard')) {
-  //   const token = await getToken({ req });
-
-  //   console.log('token', token);
+  if (req.nextUrl.pathname.startsWith('/signin')) {
+    const token = await getToken({ req });
 
-  //   if (!token) {
-  //     console.log('has no token', req.url);
-  //     const redirectUrl = new URL('/signin', req.url);
-  //     redirectUrl.searchParams.set('callbackUrl', req.url);
+    if (token) {
+      console.log('has has token', req.url);
```

Comment:
Same here as well, do we need the `console.log`? 👀

Reply:
He learns from the best (me)